### ISSUE TITLE
for fine-tune

### ISSUE THREAD

**Issue by dogdog258** [yiruoyanyu/hANMCL#1]
When i use this sentense "python train.py --dataset coco_ft --flip --net DAnA --lr 0.001 --lr_decay_step 12 --bs 2 --epochs 12 --disp_interval 20 --save_dir models/{SEED}_{SHOTS} --way 2 --shot {TRAIN SHOTS} --seed seed{SEED} --shots {SHOTS}shots --r" to fine-tune。It returns to me "usage: train.py [-h] [--dataset DATASET] [--net NET] [--flip] [--o OPTIMIZER]
                [--lr LR] [--lr_decay_step LR_DECAY_STEP]
                [--lr_decay_gamma LR_DECAY_GAMMA] [--nw NUM_WORKERS] [--ls]
                [--mGPUs] [--bs BATCH_SIZE] [--start_epoch START_EPOCH]
                [--epochs MAX_EPOCHS] [--disp_interval DISP_INTERVAL]
                [--save_dir SAVE_DIR] [--ascale ASCALE] [--eval]
                [--onc OLD_N_CLASSES] [--eval_dir EVAL_DIR] [--fs] [--way WAY]
                [--shot SHOT] [--sup_dir SUP_DIR] [--r] [--load_dir LOAD_DIR]
                [--checkepoch CHECKEPOCH] [--checkpoint CHECKPOINT] [--dlog]
                [--imlog] [--sup FT_SUP] [--seed SEED] [--shots SHOTS]
train.py: error: argument --shot: invalid int value: '{TRAIN'". How to slove it.

**Comment by yiruoyanyu** [comment]
you should make --shot {TRAIN SHOTS} the TRAIN-SHOTS to  an int ; like 10; please modify your bash code
but i dont suggest you use this project ，the structure of this project is not clear。 you can try VFA to learn few shot object detection ,the link is https://github.com/csuhan/VFA
